tools/i18n-toolkit/.i18nrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+    //path: [],
```

Comment:
Are these two commented out lines necessary? If so, please add some comment.

Reply:
Forgot to remove it .....